src/components/RiderSearch.js

```diff
@@ -0,0 +1,92 @@
+import _ from 'lodash'
+import React, { Component } from 'react'
+import { Search, Grid, Header } from 'semantic-ui-react'
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import of 'Header'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=javascript%3AS1128)